index_test.go

```diff
@@ -124,14 +124,14 @@ func TestIndex_CreateFrame(t *testing.T) {
 			}
 		})
 
-		t.Run("ErrInverseRangeNotAllowed", func(t *testing.T) {
+		t.Run("ErrInverseRangeAllowed", func(t *testing.T) {
```

Comment:
How does a frame with inverse and range enabled work? Trying to understand why it wasn't allowed before.

Reply:
I've been looking into this - the fields, and the inverse and standard views are handled separately, so the inverse setting does not affect fields. I'll add a basic test, but I'm not going to spend much time on it until we decide what we're doing with #1195

Reply:
OK, we decided to deprecate rangeEnabled - is this cool now?